src/store/modules/state.init.js

```diff
@@ -0,0 +1,7 @@
+const categoryStateInit = (name, postsPerPage, currentPage) => ({
```

Comment:
come to think of it - do we really need this function? Initially there won't be any base state related to categories since there is no default category to display, hence we won't be using this in the state initialization.

Reply:
So how to quickly replace it some file with defined categories with pages, currentPage per specific cat and pass to the {cat}.store.js some cat with params triggered by language event (default polish lang) - click e.g. flag in some corner on a top page

Reply:
that will be done by the action that will be called when user navigates to a specific URL. Please notice _maj dir frjend_ that currently this function is being called on the base state during initialization. What is the purpose of that? The user will land on the home page and then navigate to a subpage and call a request with these data (and even if they navigate directly to a specific URL, then they will call for the data anyway).